lib/git_reflow/git_server/git_hub/pull_request.rb

```diff
@@ -63,7 +63,19 @@ def comments
         end
 
         def last_comment
-          "#{comments.last.body.inspect}"
+          if comments.last.nil?
+            ""
+          else
+            "#{comments.last.body.inspect}"
+          end
+        end
+
+        def approved?
```

Comment:
This logic is incorrect. In the case that we require approvals from all commenters, we do not need to check if the last comment is an approval (as long as you have an approval from all the commenters, we do not care about the last comment). If the "last comment is a LGTM" is a piece of logic you would want to add to both cases regardless of all approval or a set number of approval, this is something we can discuss further.